07PerformanceProgramming/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Lecture 7: Performance Programming"
+---
+
+## Lecture 7: Overview
+
+### Today's Lesson
+
+- Performance Programming
+ - Why?
```

Comment:
Check indentation? Showing up as single level of indentation in the preview.

Reply:
Done